frontend/src/components/users/Grid.test.tsx

```diff
@@ -37,23 +38,23 @@ test('makeGridCoordinates', () => {
 })
 
 test.each([
-  { pos: { x: 1, y: 1 }, res: mock[4] },
-  { pos: { x: 1, y: 0 }, res: null },
-  { pos: { x: 0, y: 1 }, res: null },
-  { pos: { x: 2, y: 2 }, res: null },
+  { pos: pos(1, 1), res: mock[4] },
+  { pos: pos(1, 0), res: null },
+  { pos: pos(0, 1), res: null },
+  { pos: pos(2, 2), res: null },
 ])('moveItemSwap(mock[0], $pos, mock) -> [$res]', ({ pos, res }) =>
   expect(moveItemSwap(mock[0], pos, mock)).toEqual(res)
 )
 
 test.each([
-  { item: mock[0], pos: { x: 1, y: 1 }, res: false },
-  { item: mock[0], pos: { x: 1, y: 0 }, res: false },
-  { item: mock[0], pos: { x: 0, y: 1 }, res: false },
-  { item: mock[0], pos: { x: 2, y: 2 }, res: true },
-  { item: mock[1], pos: { x: 1, y: 0 }, res: true },
-  { item: mock[2], pos: { x: 2, y: 0 }, res: true },
-  { item: mock[3], pos: { x: 0, y: -1 }, res: true },
-  { item: mock[4], pos: { x: 1, y: 0 }, res: true },
-])('isMovableToEmpty($item, $pos, mock) -> [$res]', ({ item, pos, res }) =>
+  { item: mock[0], pos: pos(1, 1), res: false },
+  { item: mock[0], pos: pos(1, 0), res: false },
+  { item: mock[0], pos: pos(0, 1), res: false },
+  { item: mock[0], pos: pos(2, 2), res: true },
+  { item: mock[1], pos: pos(1, 0), res: true },
+  { item: mock[2], pos: pos(2, 0), res: true },
+  { item: mock[3], pos: pos(0, -1), res: true },
+  { item: mock[4], pos: pos(1, 0), res: true },
+])('isMovableToEmpty($item, $pos,, mock) -> [$res]', ({ item, pos, res }) =>
```

Comment:
```suggestion
])('isMovableToEmpty($item, $pos, mock) -> [$res]', ({ item, pos, res }) =>
```
prettier 적용 누락인 것 같습니다

Reply:
문자열 안에 있어서 그런 듯 하네요 고치겠습니다